lib/tesla_api/client.rb

```diff
@@ -1,20 +1,21 @@
 module TeslaApi
   class Client
-    include HTTParty
-    base_uri "https://owner-api.teslamotors.com/api/1"
-    format :json
+    # include HTTParty
+    # base_uri "https://owner-api.teslamotors.com/api/1"
+    # format :json
 
-    attr_reader :email, :token, :client_id, :client_secret
+    attr_reader :email, :token, :client_id, :client_secret, :headers
 
     def initialize(email, client_id = ENV["TESLA_CLIENT_ID"], client_secret = ENV["TESLA_CLIENT_SECRET"])
       @email = email
       @client_id = client_id
       @client_secret = client_secret
+      @headers = {}
```

Comment:
You don't need to initialize this. It's OK to pass `headers: nil` into HTTParty's request methods.